### ISSUE TITLE
Consider setting Consumption by default when creating a Cast activity via dropping a spell

### ISSUE THREAD

**Issue by krbz999** [foundryvtt/dnd5e#4637]
In most cases, the spell would consume uses of the parent item equal to its base level. This would save a lot of effort when creating a magical item that has a lot of spells it can cast if consumption was pre-configured.

**Comment by Fyorl** [comment]
Now that Cast activities are used frequently for NPC Spellcasting, I'm not sure this is the case anymore.

**Comment by krbz999** [comment]
> Now that Cast activities are used frequently for NPC Spellcasting, I'm not sure this is the case anymore.

I agree, it would at most be in a more limited scope. E.g. only if the parent item is a physical item.
